Cannot find module '@noreajs/common' from 'node_modules/oauth-v2-client/dist/helpers/requestTokenFunc.js'

I get this error:
```
    Cannot find module '@noreajs/common' from 'node_modules/oauth-v2-client/dist/helpers/requestTokenFunc.js'

    Require stack:
      node_modules/oauth-v2-client/dist/helpers/requestTokenFunc.js
      node_modules/oauth-v2-client/dist/helpers/revokeTokenFunc.js
      node_modules/oauth-v2-client/dist/helpers/index.js
      node_modules/oauth-v2-client/dist/index.js
      src/third-party-apis/axios-oauth2-api-client.ts

      1 | import axios, { type CreateAxiosDefaults, type AxiosInstance } from 'axios';
    > 2 | import OauthClient, { type OauthClientConfig } from 'oauth-v2-client';
```

It looks like `@noreajs/common` is only in your devDependencies, but needs to be in main dependencies?

--- a/package.json
+++ b/package.json
@@ -31,15 +31,14 @@
   "homepage": "https://github.com/noreajs/oauth-v2-client#readme",
   "dependencies": {
     "crypto-js": "^4.0.0",
-    "query-string": "^7.1.0"
+    "query-string": "^7.1.0",
+    "@noreajs/common": "^2.0.0"
   },
   "peerDependencies": {
-    "@noreajs/common": "^2.0.0",
     "@types/crypto-js": "^4.0.0",
     "axios": "^0.21.1"
   },
   "devDependencies": {
-    "@noreajs/common": "^2.0.0",
     "@types/cookie": "^0.5.1",
     "@types/crypto-js": "^4.0.0",
     "@types/express": "^4.17.8",

